metabase/metabase
# use the new endpoint to fetch compatible cards in the add series modal

Related to https://github.com/metabase/metabase/issues/28045

### Description

Uses the new endpoint from https://github.com/metabase/metabase/pull/30321 to fetch compatible cards

### How to verify

1. New dashboard -> add a question that should have compatible series to add
2. Try adding other series
3. Ensure it lists only compatible options

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30410)
